straxen/url_config/dispatcher.py

```diff
@@ -0,0 +1,85 @@
+from .regex_dispatcher import RegexDispatcher
+
+import re
+import warnings
+
+check_urls = RegexDispatcher("check_urls")
+
+
+@check_urls.register(r".*")
+def confirm_xedocs_schema_exists_check(url: str):
+    import xedocs
+    # Selects the schema name in the URL and confirms it exists in the xedocs db
+    if not re.findall(r"(?<=\://)[^\://]*(?=\?)", url)[0] in xedocs.list_schemas():
+        warnings.warn("You tried to use a schema that is not part of the xedocs database")
+
+
+@check_urls.register(r".*(list-to-array|'object-to-dict').*")
```

Comment:
only relevant to the combination of xedocs protocol and list-to-array protocol. 
also, why does object-to-dict protocol require a list?

Reply:
object-to-list expects a list as an input

Reply:
so i guess its a typo and should be "objects-to-dict"?